src/filtered_world/filtered_world.cpp

```diff
@@ -14,42 +15,43 @@ namespace rtt {
     void FilteredWorld::reset() {
 
         // Clear the input buffers
-        robots_them_buffer.clear();
-        robots_us_buffer.clear();
+        robots_blue_buffer.clear();
+        robots_yellow_buffer.clear();
 
         // These are used for the final world state
         robots_blue_world.clear();
         robots_yellow_world.clear();
         ball_world = rtt::Ball();
 
         // Initialize the input buffers.
-        robots_them_buffer = RobotMultiCamBuffer();
-        robots_us_buffer = RobotMultiCamBuffer();
+        robots_blue_buffer = RobotMultiCamBuffer();
+        robots_yellow_buffer = RobotMultiCamBuffer();
 
         // The cameras
         world_cams = std::map<int, bool>();
 
     }
 
     /// Create a message that has the world in it
-    roboteam_msgs::World FilteredWorld::as_message() const {
+    roboteam_proto::World FilteredWorld::as_message() const {
 
-        roboteam_msgs::World returnMsg;
+        roboteam_proto::World returnMsg;
         for (auto &robot : robots_blue_world) {
-            returnMsg.them.push_back(robot.second.as_message());
+            returnMsg.mutable_blue()->Add(robot.second.as_message());
         }
 
         for (auto &robot : robots_yellow_world) {
-            returnMsg.us.push_back(robot.second.as_message());
+            returnMsg.mutable_yellow()->Add(robot.second.as_message());
         }
 
-        returnMsg.ball = ball_world.as_message();
-        returnMsg.time=timeLastUpdated;
+        auto worldBall = ball_world.as_message();
+        returnMsg.set_allocated_ball(&worldBall);
```

Comment:
Might be a bit confusing since we don't use this protobuf function in other places.